config/commands/_tdd.sh

```diff
@@ -0,0 +1,39 @@
+# @yaml
+# docstring: Reproduces the current bug by running a test that was designed to fail as long as the bug exists.
+# tdd: true
+tdd_repro() {
+    # set -euo pipefail
```

Comment:
uncomment?

Reply:
We can't uncomment this and we generally cannot `set -e` because the container IPC is not setup to handle failing command calls.
I mentioned that in our convo yesterday:

* This requires some investigations, so we can actually communicate tool failures properly.
* and this is also the reason why neither the `tool_result` message nor our o11y can easily assess whether a tool call failed.